db/seeds.rb

```diff
@@ -492,6 +492,17 @@ def create_if_none(klass, count = nil)
 end
 
 ##############################################################################
+
+seeder.create_if_none(ProfileField) do
+  ProfileFields::AddBaseFields.call
+  ProfileFields::AddLinkFields.call
+  ProfileFields::AddWorkFields.call
+  ProfileFields::AddCodingFields.call
```

Comment:
I just want to clarify a follow up question that I had whilst going through the services. 

Will these specific services `AddWorkFields` and `AddCodingFields` be used only in our local development seeds or is this something we're planning on seeding the forem prod database with as well (to show up on the admin ui)?

Reply:
This is a good question. I think we can add the work fields as a preset for other Forems, but I'm not sure how useful the coding related ones will be.